week1/task_Judging the shape of a triangle.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h> 
+#include <math.h>
+#include <conio.h>
+
+#define EPS 1e-1
+
+int main ()

```

Comment:
No space between function name and (
```suggestion
int main()
```